submissions/andysmokk/dom-js/sass/layouts/_main.scss

```diff
@@ -0,0 +1,219 @@
+.main-box {
+    display: grid;
+    grid-template-columns: 1fr 3fr;
+    column-gap: 20px;
+
+    @media screen and (max-width: 580px) {
+        column-gap: 5px;
+    }
+}
+
+.list-movie-item {
+    display: flex;
+    flex-direction: column;
+}
+
+.list-movie-item:not(:last-child) {
+    margin-bottom: 10px;
+}
+
+.list-movie-btn {
+    min-height: 50px;
+    padding: 10px;
+
+    background-color: var(--background-color-btn);
+    ;
+    text-shadow: var(--default-shadow);
+    z-index: 1;
+
+    &:hover,
+    &:focus {
+        background-color: var(--hover-focus-color);
+    }
+}
+
+.list-movie-btn-text {
+    font-family: var(--default-font);
+    font-size: 12px;
+    color: var(--white-color);
+
+    text-shadow:
+        -0 -1px 0 #000000,
```

Comment:
I see you are a designer, at least in your soul)) OK let's move this beautiful var to the variables file.

Reply:
I still want to become a developer, but if that doesn't work out, I'll think about becoming a designer :grin:. Fixed.